cypress/support/fragments/data_import/match_profiles/match-profiles.js

```diff
@@ -41,17 +41,30 @@ const fillMatchProfileForm = ({
   profileName,
   incomingRecordFields,
   existingRecordFields,
-  matchCriterion
+  matchCriterion,
+  existingRecordType
 }) => {
   cy.do(TextField('Name*').fillIn(profileName));
   // select existing record type
-  cy.get('[data-id="MARC_BIBLIOGRAPHIC"]').click();
+  // TODO create waiting of upload .json files
+  cy.wait(5000);
+  cy.get(`[data-id="${existingRecordType}"]`).last().click();
   // fill MARC Bibliographic field in incoming
   fillIncomingRecordFields(incomingRecordFields);
   // choose match criterion
   cy.do(Select('Match criterion').choose(matchCriterion));
-  // fill MARC Bibliographic field in existing
-  fillExistingRecordFields(existingRecordFields);
+  if (existingRecordType === 'MARC_BIBLIOGRAPHIC') {
+    // fill MARC Bibliographic field in existing
+    fillExistingRecordFields(existingRecordFields);
+  } else if (existingRecordType === 'HOLDINGS') {
+    cy.do(Button({ id:'criterion-value-type' }).click());
+    cy.expect(SelectionList({ id: 'sl-container-criterion-value-type' }).exists());
+    cy.do(SelectionList({ id: 'sl-container-criterion-value-type' }).select('Admin data: Holdings HRID'));
+  } else {
```

Comment:
Try to avoid a lot of "if" everywhere where it possible. May here it is possible with way from my previous comment